Dockerfile

```diff
@@ -29,12 +29,12 @@ RUN \
   touch /var/log/fpm-php.www.log && \
   chown www-data:www-data /var/log/fpm-php.www.log
 
-# Install composer 1.1.1
-RUN \
-  php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
-  php -r "if (hash_file('SHA384', 'composer-setup.php') === '92102166af5abdb03f49ce52a40591073a7b859a86e8ff13338cf7db58a19f7844fbc0bb79b2773bf30791e935dbd938') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
-  php composer-setup.php --version=1.1.1 --filename=composer --install-dir=/usr/local/bin && \
-  php -r "unlink('composer-setup.php');"
+# Install composer 1.1.2  using the latest installer.
+RUN curl -o /tmp/composer-installer https://getcomposer.org/installer && \
+  curl -o /tmp/composer-installer.sig https://composer.github.io/installer.sig &&  \
```

Comment:
The reason I updated it was that the image would no longer build for that exact reason, using https://composer.github.io/installer.sig  as I do here should give us a tiny bit of security. Both sites could of cause be compromised - but it does seem a bit unlikely. Relying on the public-key has the usual problem of key-exchange - so that could be compromised as well @xendk ?